### ISSUE TITLE
[Accessibility: Resize text] Unable to manage PCI devices and snapshots when zoom 200%

### ISSUE THREAD

**Issue by alinefm** [kimchi-project/kimchi#892]
The tabs for PCI Device and Snapshot are not displayed when zoom 200%

![4cfd4a40563e6034](https://cloud.githubusercontent.com/assets/4921718/14019475/bf901f10-f1b1-11e5-8083-c5e131d07a59.jpg)
